build/vsts_build.yaml

```diff
@@ -223,6 +223,10 @@ phases:
     displayName: Verify Assembly Signatures and StrongName
     inputs:
       TargetFolder: '$(Build.Repository.LocalPath)\\artifacts\\$(NupkgOutputDir)'
+  
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
```

Comment:
is `ms.vss-governance-buildtask.governance-build-task-component-detection` necessary, or can we just use `  - task: ComponentGovernanceComponentDetection@0`?

Reply:
I think it's necessary, but I haven't tried it :)
I just followed the instructions at https://docs.opensource.microsoft.com/tools/cg.html :)